packages/next/src/server/lib/app-info-log.ts

```diff
@@ -31,7 +31,7 @@ export function logStartInfo({
   if (networkUrl) {
     Log.bootstrap(`- Network:      ${networkUrl}`)
   }
-  if (envInfo?.length) Log.bootstrap(`- Environments: ${envInfo.join(', ')}`)
+  if (envInfo?.length) Log.bootstrap(` - Environments: ${envInfo.join(', ')}`)
```

Comment:
👋🏼 Noticed this in logs, is this alignment change intentional?

```
> next build

   ▲ Next.js 15.0.4-canary.23
    - Environments: .env.production, .env # <----- Extra space indent intended?
   - Experiments (use with caution):
     · ppr

   Creating an optimized production build ...
 ✓ Compiled successfully
   Skipping validation of types
   Skipping linting
 ⚠ Using edge runtime on a page currently disables static generation for that page
 ✓ Collecting page data 
 ✓ Generating static pages (25/25)
 ✓ Collecting build traces    
 ✓ Finalizing page optimization    
```

Reply:
It is not, good catch!